src/main/java/com/uid2/shared/attest/AttestationToken.java

```diff
@@ -104,6 +144,12 @@ private static IvParameterSpec generateIv() {
         return new IvParameterSpec(iv);
     }
 
+    private static GCMParameterSpec generateGcmParam() {
+        byte[] iv = new byte[16];
```

Comment:
I believe this should 12. Compare for example uid2-operator v3 token encryption.

Reply:
good call. updated 👍